front_end/src/components/charts/multiple_choice_chart.tsx

```diff
@@ -185,6 +186,48 @@ const MultipleChoiceChart: FC<Props> = ({
       }}
     />
   );
+
+  // Define a custom "X" symbol function
+  const GetSymbol: React.FC<PointProps> = (props: PointProps) => {
+    const { x, y, datum, size, style } = props;
+    const symbol = datum.symbol;
+    const stroke = style.stroke;
+
+    if (symbol === "x") {
+      return (
+        <g>
+          <line
+            x1={x! - (size! as number)}
+            y1={y! - (size! as number)}
+            x2={x! + (size! as number)}
+            y2={y! + (size! as number)}
```

Comment:
Let's remove non-null assertion so it wouldn't crash the app in case unexpected data is received. We could guard unexpected data and return `null` as a fallback:

```
  if (
    typeof x !== "number" ||
    typeof y !== "number" ||
    typeof size !== "number"
  ) {
    return null;
  }
  ```